packages/metering/lib/processors/billing.ts

```diff
@@ -49,23 +52,69 @@ async function process(event: UsageEvent): Promise<Result<void>> {
                 }
                 const mar = event.payload.value;
                 metrics.increment(metrics.Types.BILLED_RECORDS_COUNT, mar, { accountId: event.payload.properties.accountId });
+                void trackUsage({
+                    accountId: event.payload.properties.accountId,
+                    metric: 'active_records',
+                    delta: mar
+                });
+
                 return billing.add('monthly_active_records', mar, {
                     idempotencyKey: event.idempotencyKey,
                     timestamp: event.createdAt,
                     ...event.payload.properties
                 });
             }
             case 'usage.actions': {
+                void trackUsage({
+                    accountId: event.payload.properties.accountId,
+                    metric: 'actions',
+                    delta: event.payload.value
+                });
                 return billing.add('billable_actions', event.payload.value, {
                     idempotencyKey: event.idempotencyKey,
                     timestamp: event.createdAt,
                     ...event.payload.properties
                 });
```

Comment:
We now have a queue that allows us to not lose events, so why don't we await all trackers?

Reply:
good point